x-pack/plugins/security_solution/public/common/components/visualization_actions/visualization_embeddable.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { ChartLabel } from '../../../overview/components/detection_response/alerts_by_status/chart_label';
+import type { VisualizationAlertsByStatusResponse } from '../../../overview/components/detection_response/alerts_by_status/types';
+import { useDeepEqualSelector } from '../../hooks/use_selector';
+import { inputsActions, inputsSelectors } from '../../store/inputs';
+import { DonutChartWrapper } from '../charts/donutchart';
+import { parseVisualizationData } from './utils';
+import { InputsModelId } from '../../store/inputs/constants';
+import { useRefetchByRestartingSession } from '../page/use_refetch_by_session';
+import { LensEmbeddable } from './lens_embeddable';
+import type { EmbeddableData, VisualizationEmbeddableProps } from './types';
+
+const VisualizationEmbeddableComponent: React.FC<VisualizationEmbeddableProps> = (props) => {
+  const dispatch = useDispatch();
+  const { inputId = InputsModelId.global, id, isDonut, label, onLoad, ...lensPorps } = props;
+  const { session, refetchByRestartingSession } = useRefetchByRestartingSession({
+    inputId,
+    queryId: id,
+  });
+  const getGlobalQuery = inputsSelectors.globalQueryByIdSelector();
```

Comment:
The entity analytics page doesn't have a global search bar. Therefore, if the user adds something to the global query and opens the entity analytics page, it will display inconsistent data.

<img width="1487" alt="Screenshot 2023-01-13 at 15 59 48" src="https://user-images.githubusercontent.com/1490444/212350820-6d48d680-d0cd-4c06-bf43-fa2a176a4618.png">

Reply:
Nice catch, thank you.